txbridge/src/main/java/org/jboss/jbossts/txbridge/outbound/OutboundBridgeManager.java

```diff
@@ -88,49 +93,51 @@
     /**
      * Remove the mapping for the given externalTxId. This should be called for gc when the tx is finished.
      *
-     * @param externalTxId The JTA transaction identifier.
+     * @param outboundMappingUid identifier bound to the JTA transaction
      */
-    public static synchronized void removeMapping(Uid externalTxId)
```

Comment:
As this is a public interface we can't remove the public access to this method unless it is a serious defect